Xamarin.Forms.Platform.Android/VisualElementTracker.cs

```diff
@@ -60,7 +68,13 @@ protected virtual void Dispose(bool disposing)
 				if (_renderer != null)
 				{
 					_renderer.ElementChanged -= RendererOnElementChanged;
-					_renderer.View.RemoveOnAttachStateChangeListener(AttachTracker.Instance);
+
+					if (_renderer.View.IsAlive() && _attachTracker.IsAlive())
```

Comment:
We don't need the `_context.IsDesignerContext()` check here? That is covered by `.IsAlive()`, right? Just checking

Reply:
Right I was hoping to leave the designer check out of the dispose method :-) just have as few of them peppered around as possible